app/src/main/java/edu/ucsb/cs56/projects/android/conversion_showdown/ConverterFragment.java

```diff
@@ -125,16 +125,14 @@ public void onClick(View v) {
         public void onClick(View v) {
             String input = mFromText.getText().toString();
             try {
-                //Uses Converter class to help validate input
                 Converter converter = new Converter(input, fromBase);
-                if (!converter.isValid()) {
-                    mFromText.setError("Input Format Error");
-                }
                 mToText.setText(converter.toBase(toBase));
                 MyUtils.hideSoftKeyBoard(v, inputMethodManager);
 
             } catch (IntOverFlow e) {
                 mFromText.setError("Input Number Too Large");
+            } catch (SanityCheckException e){
+                mFromText.setError("Input Format Error");
```

Comment:
Try to make same error messages. You are using both:
"Input Format Wrong" and "Input Format Error"

Reply:
Did you make these changes?

Reply:
Yes we did fix it and push to the branch proj02